.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
Out of interest, when do you need to specify dependencies directly as per https://github.com/alphagov/publishing-api/blob/main/.github/dependabot.yml?

Reply:
Sure, we do that so that only a select group of dependencies are kept up to date to avoid the high dependabot traffic of the entire tree being done.

Things are much simpler for a gem though because it doesn't have a Gemfile.lock so it doesn't need to consider the whole tree. In fact for this gem, the only dependency that can be bumped is rubocop-govuk since none of the other gems have version constraints.